app/views/companies/index.html.slim

```diff
@@ -0,0 +1,11 @@
+.container
+  .section
+    - @companies.group_by(&:supporter?).reverse_each do |supporter, companies|
```

Comment:
Ah, I get it, sorry, you're rendering the `companies` partial twice, that's why this way 🤔